c_api/concurrentqueue.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
```

Comment:
Come to think of it, this entire `#ifdef` block should be wrapped in `#ifndef MOODYCAMEL_EXPORT` for interoperability with other headers that do the same thing.